nms/app/packages/magmalte/app/views/equipment/EnodebDetailConfigEdit.js

```diff
@@ -263,8 +284,21 @@ export function RanEdit(props: Props) {
     try {
       const enb = {
```

Comment:
I might be missing something, but it seems like `enb` doesn't have a type? That should definitely be added.